<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZFieldDLg</class>
 <widget class="QDialog" name="ZFieldDLg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fieldlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZTypeLEd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Universal size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Header size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Natural size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HeaderSizeLEd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>67</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="UniversalLEd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>67</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NaturalLEd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZTypeStrLEd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="FieldNameLEd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CapacityLEd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>67</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hashcode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Key eligible</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HashcodeLEd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="KeyEligibleCHk">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ListBTn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisguardBTn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disguard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptBTn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FieldNameLEd</tabstop>
  <tabstop>ZTypeLEd</tabstop>
  <tabstop>ZTypeStrLEd</tabstop>
  <tabstop>HeaderSizeLEd</tabstop>
  <tabstop>NaturalLEd</tabstop>
  <tabstop>UniversalLEd</tabstop>
  <tabstop>CapacityLEd</tabstop>
  <tabstop>HashcodeLEd</tabstop>
  <tabstop>KeyEligibleCHk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
